.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Does this mean if a person wants to batch work together from multiple PRs before release, they'll need to land those PRs on to a non-master branch?  If so, what impact does this have on our required checks?

Reply:
hmm good question, for more clarification... do you mean about required checks at PR level? or when pushing to `master`?

Reply:
I originally thought the release process was automatically like it is with semantic-release, but that isn't the case with changesets because someone needs to create a release PR.

Reply:
I chatted with Juan in Slack and learned that the action runs once to generate the PR if there's any changeset markdown files.  This PR converts those markdown files to changelogs.  Once the PR lands then the action runs a second time and will do the publish since there are no pending changeset markdown files.